src/services/dashboards/dashboard-detail/lib/config.ts

```diff
@@ -0,0 +1,21 @@
+import lodash from 'lodash';
+
+//
+// If the RANGE varies for each size, the last element must be added up to the length of the largest RANGE among sizes.
+// e.g.) Added 540, 620 in SM
+// [320, 400, 480, 560, 640]
+// [480, 560, 640, 640, 640]
+// [640, 720, 800, 800, 800]
+// [800, 880, 960, 960, 960
+
+export const WIDGET_FRAME_WIDTH_RANGE_LIST = [
+    [320, 400, 480], // 0: SM
+    [480, 560, 640], // 1: MD
+    [640, 720, 800], // 2: LG
+    [800, 880, 960], // 3: XL
+];
```

Comment:
It doesn't seem to have to be an array, how about changing it like this?
Then it seems that you don't have to search by number index in `widgetFrameSizeRangeExtractor()`.

```suggestion
export const WIDGET_FRAME_WIDTH_RANGE = {
    SM: [320, 400, 480],
    MD: [480, 560, 640],
    LG: [640, 720, 800],
    XL: [800, 880, 960],
};
```

---
꼭 array일 필요는 없어 보이는데 이렇게 바꾸는 건 어떨까요?
그럼 widgetFrameSizeRangeExtractor에서  number index로 찾지 않아도 될 것 같습니다.

Reply:
Great! I'll apply it. Thanks a lot